ietf-optical-resource-pm-types.yang

```diff
@@ -55,90 +55,96 @@ module ietf-optical-resource-pm-types {
     'MAY', and 'OPTIONAL' in this document are to be interpreted as
     described in BCP 14 (RFC 2119) (RFC 8174) when, and only when,
     they appear in all capitals, as shown here.";
-               
-  
+
+
   revision 2023-07-04 {
     description  "Initial revision.";
-    
+
   }
 
   identity resource-type {
     description "this is the base type of all the rerource type";
   }
-  
+
   identity network-element {
     base resource-type;
-	description "NE resource type";
+        description "NE resource type";
   }
-  
+
+    identity interface {
+        base resource-type;
+        description “Network interface”;
+	}
+
+
   identity board {
     base resource-type;
-	description "board resource type";
+        description "board resource type";
   }
-  
+
   identity termination-point {
     base resource-type;
-	description "Termination point resource";
+        description "Termination point resource";
   }
-  
+
   identity tunnel-termination-point {
     base resource-type;
-	description "Tunnel termination point resource";
-  }  
-  
+        description "Tunnel termination point resource";
+  }
+
   identity period {
-    description 
-	  "this is the base type of all the performace monitoring priod 
-	  type.";
+    description
+          "this is the base type of all the performace monitoring priod
+          type.";
   }
-  
+
   identity period-15-minutes {
     base period;
-	description 
-      "the during of monitoring task will be repeated at every 15 
-	  minutes";
+        description
+      "the during of monitoring task will be repeated at every 15
+          minutes";
   }
-  
+
   identity period-24-hours {

```

Comment:
Same comment. We should use draft https://datatracker.ietf.org/doc/draft-ietf-netmod-schedule-yang/. 

You can have a look how I use it in https://github.com/vlopezalvarez/draft-contreras-opsawg-scheduling-oam-tests/blob/main/Yang/ietf-oam-unitary-test.yang. I can change the YANG model if you agree